### PR TITLE
Changes to basic layout and CSS

### PR DESCRIPTION
**Pull request by astolarz**
I've added a couple extra divs (and CSS rules) to fit the page within a certain width (currently set at 980px). 980px seems to be the standard width (<a href="http://webplantmedia.com/web-developing/2010/04/what-width-should-i-make-my-website/">here's a possibly outdated list of page widths</a>). Facebook uses 981px it seems (perhaps it changes based on the screen?), but it also has a chat bar on the right hand side. I think this would be a good point of debate.

I've also added a rule to remove text-decoration from links, and made header links white.

Divs added:
orange_bar (around header)
content (around main)

### CODE REVIEW

**Comment by amshali** [comment]
As soon as you change the width and send another commit I will merge this. Thanks.

File: public/stylesheets/main.css [hunk 1]
```diff
@@ -19,9 +23,23 @@ body {
...
-#header {
+#orange_bar {
 	background-color: #CC5500;
-  padding: 0;
+}
+
+#header {
+	margin: auto;
+	padding: 2px 0 2px 0;
+	width: 1040px;
```
Comment (amshali) [inline public/stylesheets/main.css:33]: Can you fix the width?

**Comment by ProCynic** [comment]
howbout we just use the header tag?

**Comment by amshali** [comment]
Please fix the width and recommit.

**Pull request closed without merging**